interfaces/builtin/camera.go

```diff
@@ -34,11 +36,65 @@ const cameraConnectedPlugAppArmor = `
 /run/udev/data/c81:[0-9]* r, # video4linux (/dev/video*, etc)
 `
 
-// NewCameraInterface returns a new "camera" interface.
-func NewCameraInterface() interfaces.Interface {
-	return &commonInterface{
-		name: "camera",
-		connectedPlugAppArmor: cameraConnectedPlugAppArmor,
-		reservedForOS:         true,
+var cameraPermanentSlotUdev = []byte(`
+# RealSense UVC cameras (R200, F200, SR300 LR200, ZR300)
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a80", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0a66", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0aa5", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0abf", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0acb", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="0ad0", MODE:="0666", GROUP:="plugdev"
+SUBSYSTEMS=="usb", ATTRS{idVendor}=="8086", ATTRS{idProduct}=="04b4", MODE:="0666", GROUP:="plugdev"
```

Comment:
Not that the acl package is installed in core (see https://bugs.launchpad.net/snappy/+bug/1646144), I think you can use 0660 and video. If realsense is v4l, then 50-udev-default.rules should add it automatically to the 'video' group and 70-uaccess.rules should add it to uaccess, which means you shouldn't need GROUP at all. Please check if you need MODE (if you do, set it to 0660). With this and the uaccess rule, this should work consistently with how other usb cameras are setup: root and console users can access it. non-root, non-console users would need something along the lines of what I put forth in https://bugs.launchpad.net/snappy/+bug/1646144, but right now that is an unsupported corner case in snappy (services in snappy always run as root), so fixing that shouldn't block this PR.